components/steering_wheel/SConscript

```diff
@@ -229,6 +229,10 @@ objs.extend(compile_objects(env, project_source_files))
 
 ## done compiling
 
+doc = GetOption("doc")
+if doc:
+    Execute(f"doxygen doxygen_stw.conf")
```

Comment:
seems like you added a doxygen tool to `site_scons` in your other PR, why aren't you using it here?

Also, this should really be an alias rather than an execute
`Alias("docs", env.dox())` replace second parameter with the correct call into the tool you added

Reply:
I actually wanted to talk about this with you. When I had that as the implementation (Why the dox.py exists), it would always execute, so I am a bit stuck with that there where it doesnt really make sense. Could you look into that and advise, as I am less familiar with SCons. This was the only method where it worked appropriately